tests/tasks/test_linkstatetask.py

```diff
@@ -239,15 +240,15 @@ async def test_when_timeout_occurs_poll_single_interface_should_not_crash(self,
 def linkstatetask_with_links_up(snmpsim, snmp_test_port):
     device = PollDevice(name="buick.lab.example.org", address="127.0.0.1", port=snmp_test_port, community="linksup")
     state = ZinoState()
-    task = LinkStateTask(device, state)
+    task = LinkStateTask(device, state, zino.state.config)
     yield task
 
 
 @pytest.fixture
 def linkstatetask_with_one_link_down(snmpsim, snmp_test_port):
     device = PollDevice(name="buick.lab.example.org", address="127.0.0.1", port=snmp_test_port, community="linksdown")
     state = ZinoState()
-    task = LinkStateTask(device, state)
+    task = LinkStateTask(device, state, zino.state.config)
```

Comment:
Why does this task need the global config variable during tests?  Won't an empty (i.e. default values only) `Configuration` object do?